core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -425,8 +434,25 @@ private void tryStart(Instant startedAt) {
             // For all registered output consumers, start following as close to container startup as possible
             this.logConsumers.forEach(this::followOutput);
 
+            // Wait until inspect container returns the mapped ports
+            Callable<Integer> inspectMappedExposedPortsCount = () -> {
+                containerInfo = dockerClient.inspectContainerCmd(containerId).exec();
+                return (int) containerInfo
+                    .getNetworkSettings()
+                    .getPorts()
+                    .getBindings()
+                    .values()
+                    .stream()
+                        .filter(Objects::nonNull)
+                        .count();
```

Comment:
indentation issue

Reply:
You don't want additional indent for the stream operations?

Reply:
There are no "stream operations" (and IDEA would not indent it like this, btw, unless this is something new :)), and overall it looks very inconsistent :)

We do sometimes (where the line length permits) use something like:
```java
user.getProfile().getFriends().stream()
    .filter(User::isAdult)
    .count();
```

but there is no need to extra indent methods of `Stream` :)